PLAYBOOKS.md

```diff
@@ -10,44 +10,21 @@ A playbook is a JSON file that will be used by our [scraping engine](./app/engin
-      // icao codes. We select a selector on basis of icao code.
-      "chart": {
-        "baseUrl": "${baseUrl}/eAIP/",
-        "selector": "a[title=\"${icao}\"]",
-        "attribute": "href"
-      }
-    }
-  }
-}
-```
+| Keys          | Description                                            | Data type        | Required | Parent key | Example                                                                                                                          |
+| ------------- | ------------------------------------------------------ | ---------------- | -------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| iso           | ISO code of country                                    | String           | Yes      | Country    | `"iso":"IN"`                                                                                                                     |
+| name          | Name of the country                                    | String           | Yes      | Country    | `"name": "India"`                                                                                                                |
+| source        | Source of information                                  | String           | Yes      | source     | `"source": "Airports Authority of India"`                                                                                        |
+| engine        | Name of scraping engine(Axios currently)               | String           | Yes      | scraper    | `"engine": "axios"`                                                                                                              |
+| baseUrl       | URL for charts website(eAIP)                           | String           | Yes      | features   | `"baseUrl": "https://aim-india.aai.aero/eaip-v2-02-2021"`                                                                        |
+| paths         | Path for getting charts from eAIP                      | Array            | Yes      | features   | [See example](https://github.com/drph4nt0m/avbot-charts/blob/ec58c14ae01da70bcff68a4360027a2142a80366/playbooks/AU.json#L11-L15) |
+| route         | Route for the front page                               | String           | Yes      | paths      | `"route": "/index-en-GB.html"`                                                                                                   |
+| navigations   | Navigations for the engine to get charts page          | Array            | Optional | paths      | [See example](https://github.com/drph4nt0m/avbot-charts/blob/ec58c14ae01da70bcff68a4360027a2142a80366/playbooks/IN.json#L14-L23) |
+| chart.baseUrl | URL for charts page from eAIP                          | String           | Yes      | chart      | [See example](https://github.com/drph4nt0m/avbot-charts/blob/ec58c14ae01da70bcff68a4360027a2142a80366/playbooks/IN.json#L26-L30) |
```

Comment:
Not exactly the url for charts page but just a prefix for the value we get from href using the selector